docs/CONTRIBUTING.md

```diff
@@ -19,33 +19,128 @@ Please submit pull requests in response to open issues. If you have a bug or fea
 
 ## Environment Setup
 
-**Prerequisites:**
+### Prerequisites:
 
 - [Node.js (npm)](https://nodejs.org/en/download/)
-- [Redis](https://redis.io/download)
+- Redis (2 methods)
+  - Use [Docker and docker-compose](#docker-and-docker-compose-set-up)
+  - Install as a [native application](#using-redis-as-a-native-application)
 
-**Redis Set Up**
-Some helpful guides:
+### Using Redis as a native application
 
-- [Redis for Linux](https://redis.io/download#installation) (recommended but not required)
-- [Redis for Windows Subsystem for Linux](https://anggo.ro/note/installing-redis-in-ubuntu-wsl/) (not recommended by Redis)
-- Redis for Windows - You can use [this](https://github.com/tporadowski/redis/releases) installer to get up and running quickly for now.
-- Redis for MacOS - need instructions
-  An easier solution would be to use Docker.
+#### Linux:
 
-**Setup**
+- Install Redis using your distribution's package manager, for example:
```

Comment:
I suggest either removing this first bullet point `- `, or increasing the indentation of the following two.

Reply:
Done.